k2/csrc/intersect_dense_pruned.cu

```diff
@@ -403,11 +320,9 @@ class MultiGraphDenseIntersectPruned {
         c_, num_seqs_, lambda_set_final_and_final_t, (int32_t i)->void {
           int32_t b_chunk_size =
               b_fsas_row_splits1[i + 1] - b_fsas_row_splits1[i];
-          int32_t final_t = final_t_data[i];
-          final_t =
-              is_final ? final_t + b_chunk_size : final_t + b_chunk_size - 1;
-          final_t_data[i] = final_t;
+          final_t_data[i] = his_t + b_chunk_size - 1;
```

Comment:
perhaps you mean this_t?

Reply:
I mean history_t,  this_t may be a better name, or start_t

Reply:
oh, history_t is fine.
google style guide prefers using whole words, not shortened ones.